tools/ci/azure/safari-technology-preview.rb

```diff
@@ -1,10 +1,10 @@
 cask 'safari-technology-preview' do
-  if MacOS.version <= :mojave
-    version '108,001-15210-20200610-d35e22e0-d9fa-4503-9988-cf7b2b554e15'
-    sha256 '0299fd2f2836b170a8b633b00d289bcc6913716042e82251af4fa1d5394b87d5'
+  if MacOS.version <= :catalina
```

Comment:
I'm getting confused by what this if switch does.

Reply:
Not sure, it's from the automated autofoolip script. I just copy/pasted.